dts/arm/nordic/nrf51822.dtsi

```diff
@@ -63,6 +63,7 @@
 			reg = <0x40000000 0x1000>;
 			interrupts = <0 1>;
 			status = "ok";
+			label = "CLOCK_0";
```

Comment:
I think we could use just "CLOCK" as the label, as we use "WDT", for example.

Reply:
Community: What you prefer?

Reply:
No opinion, but @anangl has the better overview so I would go with his judgement

Reply:
CLOCK is ok

Reply:
Changed.